benchmarks/src/main/java/com/google/samples/apps/nowinandroid/foryou/ForYouActions.kt

```diff
@@ -21,15 +21,17 @@ import androidx.test.uiautomator.By
 import androidx.test.uiautomator.Until
 import androidx.test.uiautomator.untilHasChildren
 import com.google.samples.apps.nowinandroid.flingElementDownUp
+import org.junit.Assert.fail
 
 fun MacrobenchmarkScope.forYouWaitForContent() {
     // Wait until content is loaded by checking if topics are loaded
     device.wait(Until.gone(By.res("loadingWheel")), 5_000)
     // Sometimes, the loading wheel is gone, but the content is not loaded yet
     // So we'll wait here for topics to be sure
-    val obj = device.findObject(By.res("forYou:topicSelection"))
-    // Timeout here is quite big, because sometimes data loading takes a long time!
-    obj.wait(untilHasChildren(), 60_000)
+    device.findObject(By.res("forYou:topicSelection"))?.run {
+        // Timeout here is quite big, because sometimes data loading takes a long time!
+        wait(untilHasChildren(), 30_000)
+    }
```

Comment:
I'm not sure that we really should silence (ignore) `forYou:topicSelection` not being present for the profile generation.

Reply:
We can consider changing it back in the future, but for now it's acceptable.